views/update/confirm-overseas-entity-details.html

```diff
@@ -0,0 +1,87 @@
+{% extends "layout.html" %}
+
+{% set title = "Confirm Overseas Data" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {% include "includes/list/errors.html" %}
+    <h1 class="govuk-heading-xl">Confirm this is the registered overeseas entity (NOT LIVE)</h1>
+
+  {% from "govuk/components/table/macro.njk" import govukTable %}
+  {{ govukTable({
+    captionClasses: "govuk-table__caption--m",
+    firstCellIsHeader: true,
+    rows: [
+      [
+        {
+          text: "Overseas Entity Name",
+          attributes: {
+            "id": "oe_entity_name"
+          }
+        },
+        {
+          text: appData.entity.name,
+          attributes: {
+            "id": "oe_entity_name_value"
+          }
+        }
+      ],
+      [
+        {
+          text: "Registered Office Address",
+          attributes: {
+            "id": "oe_entity_registered_address"
+          }
+        },
+        {
+          text: "" if appData.entity.name === undefined else appData.entity.principal_address.line_1 + " " + appData.entity.principal_address.country,
+          attributes: {
+            "id": "oe_entity_address_value"
+          }
+        }
+      ],
+      [
+        {
+          text: "Registered On",
+          attributes: {
+            "id": "oe_entity_registered_date"
```

Comment:
This can be an additional thing we pass in from the session. Stored separately from the Entity.

Reply:
We can discuss the need for this field separately